src/md/verification.md

```diff
@@ -0,0 +1,82 @@
+# How to verify Apache Apex (incubating) release candidate builds
+
+## Prerequisites
+1. *gpg* program should be installed on your system.
+2. Download the KEYS file and import it. This is one time activity.
+```bash
+wget https://dist.apache.org/repos/dist/release/incubator/apex/KEYS
+gpg --import KEYS
+```
+3. You can also created your own key which is required if you would like to sign the build. This step is optional.
+```bash
+gpg --gen-key
+```
+Please provide 4096 as keysize while generating the key.
+
+## File integrity check
+Download all files present in staging directory of the RC build. Staging directory link is shared in VOTE thread of the release candidate.
+
+```bash
+wget -r -np -nd <staging-area-link>/
+```
+Note the link should end with "/".
+
+Define the apex release candidate variable. We will set it up *apex-3.3.0-incubating* as an example.
+```bash
+APEX_RELEASE_CANDIDATE=apex-3.3.0-incubating
+```
+
+Verify integrity of tar.gz file:
+```bash
+gpg --verify $APEX_RELEASE_CANDIDATE-source-release.tar.gz.asc
```

Comment:
http://www.apache.org/info/verification.html mentions additional step for validating authenticity of a key. Should we add this step?